Concordium Client release March 2023 (#764)
Concordium/concordium.github.io: source/mainnet/net/references/concordium-client.rst

through a :ref:`local node<run-a-node>`. If the node runs on a different machine
or in a custom setup, the options ``--grpc-ip`` and ``--grpc-port`` can be used
to set the `IP address`_ and `port number`_ that the node is accessible at. These

> flags are supported by all ``concordium-client`` commands. Note that as of version

Reviewer: Me neither. It is mentioned in the `usage` section of the readme IIRC.

> flags are supported by all ``concordium-client`` commands. Note that as of version

Reviewer 2: Since we haven't had the need to use it yet, it is probably not important enough to be stated here. The readme is enough.

> flags are supported by all ``concordium-client`` commands. Note that as of version

Reviewer 3: You have to have a very specific setup for that to make sense. We should not mention it.

> 5.1.0, the IP address must belong to a node running with the node GRPC API V2 enabled
> and served on the specified port. For earlier versions, the IP address must belong to
> a node running with the node GRPC API V1 enabled and served on the specified port.

Reviewer 3: ```suggestion
5.1.1, the `port number`_ must be the port where the GRPC V2 interface is enabled, in contrast to previous versions which required the port number of the V1 API of the concordium node.
```

Run Concordium Client from the command line. On MacOS or Linux, access the command line with the Terminal application. On Windows, use the Power Shell or Command Prompt application. If you run it outside of the command line (e.g., by double clicking in Windows Explorer), then the Concordium Client will exit immediately without doing anything useful.

> To run the Concordium Client, you have to specify its full path unless you are in the same directory. If using MacOS the installation puts it in $PATH so that you can type ``concordium-client`` from anywhere. You must specify the file name, including the version number. For example, assuming that you saved the ``concordium-client_5.1.0-0.exe`` in the Downloads folder of a user called User, then the full path is probably ``C:\Users\User\Downloads\concordium-client_5.1.0-0.exe``. So you enter the full path at the prompt in the terminal.

Reviewer 3: ```suggestion
To run the Concordium Client, you have to specify its full path unless you are in the same directory. If using MacOS the installation puts it in $PATH so that you can type ``concordium-client`` from anywhere. You must specify the file name, including the version number. For example, assuming that you saved the ``concordium-client_5.1.1.exe`` in the Downloads folder of a user called User, then the full path is probably ``C:\Users\User\Downloads\concordium-client_5.1.1.exe``. So you enter the full path at the prompt in the terminal.
```

When running commands for the Concordium Client in the terminal, replace concordium-client with ``C:\Users\User\Downloads\concordium-client_<version>.exe`` as in the following example:

.. code-block:: console

>    C:\Users\User\Downloads\concordium-client_5.1.0-0.exe config account import concordium-backup.concordiumwallet --name AccountA

Reviewer 3: ```suggestion
   C:\Users\User\Downloads\concordium-client_5.1.1.exe config account import concordium-backup.concordiumwallet --name AccountA
```

.. Note::

   To import the backup file as shown in the example, you must be in the same directory where the concordium-backup.concordiuwallet is saved. If not, you have to specify the full path to the file, for example:

   .. code-block:: console

>       C:\Users\User\Downloads\concordium-client_5.1.0-0.exe config account import C:\Users\User\Desktop\concordium-backup.concordiumwallet --name AccountA

Reviewer 3: ```suggestion
      C:\Users\User\Downloads\concordium-client_5.1.1.exe config account import C:\Users\User\Desktop\concordium-backup.concordiumwallet --name AccountA
```
